manager/manager.go

```diff
@@ -226,6 +228,7 @@ type manager struct {
 	ignoreMetrics            container.MetricSet
 	containerWatchers        []watcher.ContainerWatcher
 	eventsChannel            chan watcher.ContainerEvent
+	httpClient               *http.Client
```

Comment:
I think you should call this `collectorHTTPClient`, since the flags are specifically specifying collector certs.